documents/policies/internal/Contractor Compliance.md

```diff
@@ -0,0 +1,30 @@
+# WCA Contractor Compliance Policy
+
+### Version 1.0 {.version}
+
+## Purpose
+The purpose of this policy is to define how the WCA handles contractors who provide services for the organization in return for monetary benefit. 
+
+## Policy
+1. The Board in its absolute discretion has the ability to determine the need for the WCA to seek assistance from a contractor in delivering key services for the functioning of the organization
+   1. Team or Committee leaders may approach the Board with a proposal to have part of the duties of their team or committee be handled by a contractor
+2. Where the need for a contractor is established the Board shall establish a subcommittee of WCA staff responsible for the management and oversight of the work to be carried out.
+   1. The subcommittee may include but is not limited to:
+      1. At least one Board member;
+      2. The leader of the relevant Team or Committee (if any);
+         1. If the leader is unable to participate on the subcommittee due to a conflict of interest they may be replaced by a Senior Member of the Team or Committee
```

Comment:
If this is "may include but not limited to" then it seems like there are no requirements for who is on the subcommittee, other than members being WCA Staff. In that case, this whole section has no purpose.

If a Board member is required, or a committee leader for whom the contractor will be working with, that should be more explicitly declared.